src/menu/option-list.js

```diff
@@ -29,6 +29,7 @@ export default function OptionList({
     StyledListItem,
   );
   const sharedProps = {
+    $disabled: item.disabled,
```

Comment:
This should be mapped on a level up. Current fix only handles list type options but, for example, misses provided profile type options. Look into `getSharedProps` called from the `/menu.js`.

Reply:
Gotcha, will update